src/Command/DebugBlocksCommand.php

```diff
@@ -18,16 +18,28 @@
 use Symfony\Component\Console\Output\OutputInterface;
 use Symfony\Component\OptionsResolver\OptionsResolver;
 
-class DebugBlocksCommand extends BaseCommand
+/**
+ * @final since 4.0
+ *
+ * NEXT_MAJOR: Comment out the "final" class declaration
```

Comment:
I think you mean the opposite :wink: 

https://english.stackexchange.com/questions/33483/when-i-say-comment-out-does-it-mean-to-uncomment-something-or-comment-it/33502
```suggestion
 * NEXT_MAJOR: Uncomment the "final" class declaration
```